grammars/c#.cson

```diff
@@ -435,6 +442,18 @@
         'include': '#code'
       }
     ]
+  'genericParameters':
+    'begin': '<(in|out)?\\s*\\b([\\w.\\[\\]]+)'
+    'beginCaptures':
+      '1':
+        'name': 'storage.type.generic.variance.cs'
+      '2':
+        'name': 'storage.type.generic.arg.cs'
+    'end': '(?:(,)|(?=[>]))'
```

Comment:
not sure why this pattern only seems to catch the first generic param... shoudl work the same way as `#parameters`, which seems to be applied for each param in a method signature. but it doesn't...